### PR TITLE
[base] [coding] Moved BWT and MTF from base/ to coding/.

### PR DESCRIPTION
**Pull request by mpimenov**
Follow-up to https://github.com/mapsme/omim/pull/6462

### CODE REVIEW

**Comment by mpimenov** [comment]
JTALL

File: base/CMakeLists.txt [hunk 1]
```diff
@@ -11,8 +11,6 @@ set(
   bidirectional_map.hpp
   bits.hpp
   buffer_vector.hpp
-  bwt.cpp
-  bwt.hpp
```
Comment (tatiana-yan) [inline base/CMakeLists.txt]: в xcode не надо перенести?

Comment (mpimenov) [reply base/CMakeLists.txt]: done

**Comment by mpimenov** [comment]
JTALL

**Pull request merged**